extensions/keycloak-authorization/runtime/src/main/java/io/quarkus/keycloak/pep/runtime/VertxHttpFacade.java

```diff
@@ -107,7 +108,15 @@ public Cookie getCookie(String cookieName) {
 
             @Override
             public String getHeader(String name) {
-                return request.getHeader(name);
+                String value = request.getHeader(name);
+
+                if (name.equalsIgnoreCase(HttpHeaders.CONTENT_TYPE.toString())) {
+                    if (value.indexOf(';') != -1) {
+                        return value.substring(0, value.indexOf(';'));
+                    }
+                }
+
+                return value;
```

Comment:
What's the reason of this change? It looks a bit unrelated and unexpected.

Reply:
That was part of @pedroigor original approach, I don't think it is needed so I might modify the PR to just keep the tests and not the rest of the changes.

Reply:
Yeah, the change is not related with this one but to #5959

Reply:
OK, then this needs proper documentation with a TODO pointing to a GH issue so that we remove it in the future.